assets/facets.js

```diff
@@ -120,7 +120,11 @@ class FacetFiltersForm extends HTMLElement {
     FacetFiltersForm.renderActiveFacets(parsedHTML);
     FacetFiltersForm.renderAdditionalElements(parsedHTML);
 
-    if (countsToRender) FacetFiltersForm.renderCounts(countsToRender, event.target.closest('.js-filter'));
+    if (countsToRender) {
+      const closestFilterElID = event.target.closest('.js-filter').id;
+      FacetFiltersForm.renderCounts(countsToRender, event.target.closest('.js-filter'));
+      FacetFiltersForm.renderDrawerCounts(countsToRender, document.getElementById(closestFilterElID));
```

Comment:
Maybe im missing something 🤔 Why do we need to add `renderDrawerCounts`. Doesnt `renderCounts` account for drawer, vertical and horizontal? Shouldnt this be working currently?

Reply:
But drawer desktop already existed before the introduction of `and` and `or`. Does that make sense?

Reply:
I could merge the logic all under `renderCounts`, I don't have a strong opinion here, but it will make the function more complex.